ipam/ring/ring_test.go

```diff
@@ -506,6 +506,22 @@ func TestOwner(t *testing.T) {
 
 }
 
+func makePeerName(i int) mesh.PeerName {
+	peer, _ := mesh.PeerNameFromString(fmt.Sprintf("%02d:%02d:00:00:00:00", i/10, i%10))
```

Comment:
Actually, we aren't going to run with more than 100 peers, are we? So I suggest we just make this `fmt.Sprintf("%02d:...:ff", i)` (the `ff` is there because all-zeros is treated specially in some parts of mesh/weave), but also add a panic for when i>100 (so that if anybody changes the numPeers in the tests they get an obvious explosion).